src/components/layout/LocaleSelector.vue

```diff
@@ -0,0 +1,206 @@
+<template>
+	<div class="navigation-locale" :class="{ 'active': active }" ref="localeSelectorRef">
+		<button
+			class="navigation-locale-toggle"
+			@click="active = !active"
+			@keyup.esc="active = false"
+			@blur="handleLocaleItemBlur"
+			aria-controls="navigation-locale-dropdown"
+			:aria-label="$t( active ? 'aria_locale_toggle_label_open' : 'aria_locale_toggle_label_closed' )"
```

Comment:
If we ever check our translation keys this will be one of the places where we trip up the check tool. We could avoid this by doing `active ? $t(key1) : $t(key2)` instead. But this is no blocker.

Reply:
Good point, I forgot about our language use checker. I fixed it.